frontend/src/metabase/dashboard/containers/DashboardHeader.jsx

```diff
@@ -297,11 +304,11 @@ class DashboardHeader extends Component {
         </span>,
       );
 
-      if (canEdit) {
+      if (canEdit && hasModelActionsEnabled) {
         buttons.push(
           <>
             <DashboardHeaderActionDivider />
-            <Tooltip key="add-action-button" tooltip={t`Add action button`}>
+            <Tooltip key="add-action-button" tooltip={t`Add`}>
```

Comment:
Why are we updating a tooltip here?

Reply:
<insert expletive here> that's not great on my part, I updated that to make sure I was working on the right part of the code. Thanks for the catch